atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraClusterOperations.java

```diff
@@ -58,6 +60,10 @@ public void replicateSystemAuthenticationDataOnAllNodes()
         }
     }
 
+    public void disableAutoCompaction() throws IOException, InterruptedException {
+        runNodetoolCommandOnAllContainers("disableautocompaction", NODETOOL_REPAIR_TIMEOUT_SECONDS);
```

Comment:
Interesting, is this a problem only when multinode tests are running? If not should we run this command on the docker image https://hub.docker.com/r/palantirtechnologies/docker-cassandra-atlasdb/~/dockerfile/ add a line `CMD ["nodetool disableautocompaction"]`.?

Reply:
+1 to Himangi's question

Reply:
a) the suggested command failed :(
b) This PR disproved that it's only a multinode problem - a build failed because of the heartflake on a single-node container!